covins_backend/src/covins_backend/optimization_be.cpp

```diff
@@ -830,13 +830,20 @@ auto Optimization::OptimizeRelativePose(KeyframePtr kf1, KeyframePtr kf2, Landma
     return numCorrespondences - numBad;
 }
 
-auto Optimization::PoseGraphOptimization(MapPtr map, PoseMap corrected_poses)->void {
+auto Optimization::PoseGraphOptimization(
+    MapPtr map, PoseMap corrected_poses) -> void {
+  
+    // std::cout << "+++ PGO: Start +++" << std::endl;
 
     ceres::Problem::Options problem_options;
     problem_options.enable_fast_removal = true;
     ceres::Problem problem(problem_options);
 
-    ceres::LocalParameterization *local_pose_param = new robopt::local_param::PoseQuaternionLocalParameterization();
+    ceres::LossFunctionWrapper *loss_function = new ceres::LossFunctionWrapper(
+        new ceres::CauchyLoss(0.5), ceres::TAKE_OWNERSHIP);
```

Comment:
Umm not sure but all the results that we reported were using these params. The low value was somehow due to the weights that we provided so I would recommend keeping it as is

Reply:
But that in return would mean that the weights that are used are generally too low. If the results are obtained by this value we can leave it. Ideally this value would be a parameter that can be configured. Because in the end it depends on the weights that are used, which (partially) are parameters themselves.

Reply:
Adding a parameter seems a good way. I will do that